tests/imap/MailCollector.php

```diff
@@ -593,12 +596,16 @@ public function testCollect() {
       // Check creation of expected documents
       $expected_docs = [
          '00-logoteclib.png',
-         '01-Screenshot-2018-4-12 Observatoire - France très haut débit.png',
+         // Space is missing between "France" and "très" due to a bug in laminas-mail
+         '01-Screenshot-2018-4-12 Observatoire - Francetrès haut débit.png',
```

Comment:
@edgardmessias 

Did you saw that ?

I guess this is only a side effect of your fix. Indeed, now the space char is located at the beginning or end of a param line, and trimming done in Laminas removes it. Before your fix, this space was certainly located in the middle of second parameter line.

Param lines are extracted with these values:
```
array(2) {
  [0]=>
  string(45) "01-Screenshot-2018-4-12 Observatoire - France"
  [1]=>
  string(21) "très haut débit.png"
}
```